### PR TITLE
Fixes Bicaridine OD not properly healing internal bleeding, This Time I Know It's For Real Edition

### PR DESCRIPTION
**Pull request by 9600bauds**
https://www.youtube.com/watch?v=DCtDAAPO-j4
thistime4
https://www.youtube.com/watch?v=DCtDAAPO-j4
thistime2
https://www.youtube.com/watch?v=DCtDAAPO-j4
thistime3
https://www.youtube.com/watch?v=DCtDAAPO-j4
thistime1
https://www.youtube.com/watch?v=DCtDAAPO-j4
thistime5
https://www.youtube.com/watch?v=DCtDAAPO-j4

:cl:
 * bugfix: Fixed Bicaridine OD not properly healing internal bleeding, for real this time.

### CODE REVIEW

**Comment by Probe1** [comment]
>hey our ODs are shit and people walk around with 1000u of anti toxin cause lel no OD
 >lets make it worse.

i'm on to you casuals

**Comment by 9600bauds** [comment]
But bicaridine OD DOES cause toxin damage

**Comment by Probe1** [comment]
Which is nullified by anti toxin not having an appreciable OD and a laughably weak OD on its own.

Bicard ODs should CAUSE internal bleeding not prevent it.  I'M COMING FOR YOU MEDBAY.

**Comment by Probe1** [comment]
Still slightly joking not totally joking jokes aside I'm not a massive knob and I'm not demanding we fix everything wrong with the game before agreeing to a bug fix ok lets go merge after review.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -3070,7 +3070,7 @@
 			M.adjustToxLoss(1)
 			M.Dizzy(5)
 			M.Jitter(5)
-			if(prob(5))
+			if(prob(5) && M.feels_pain())
```
Comment (gbasood) [inline code/modules/reagents/Chemistry-Reagents.dm:3073]: why is this here

Comment (9600bauds) [reply code/modules/reagents/Chemistry-Reagents.dm:3073]: because I'm too good to be atomic

Comment (Probe1) [reply code/modules/reagents/Chemistry-Reagents.dm:3073]: Not good practices.  You set an example for us all.
